modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -31,106 +30,101 @@ https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html[NGINX].
 5.  xref:create-an-ssl-certificate[Create an SSL certificate]
 6.  xref:web-server-setup[Web Server setup]
 7.  xref:test-the-setup[Test the setup]
-8.  xref:renewing-certificates[Renewing Certificates]
+8.  xref:renewing-ssl-certificates[Renewing SSL Certificates]
+9.  xref:deleting-ssl-certificates[Deleting SSL Certificates]
 
 [[requirements-dependencies]]
 == Requirements & Dependencies
 
-* You require a domain name with a valid
-https://support.dnsimple.com/articles/a-record/[A record] pointing back
-to your servers IP address. In case your server is behind a firewall,
-take the necessary measures to ensure that your server is accessible,
-worldwide, from the internet, by adding the required firewall and port
-forward rules.
+You require a domain name with a valid
+https://support.dnsimple.com/articles/a-record/[A-Record] pointing back
+to your servers IP address. In case your server is behind a firewall, take
+the necessary measures to ensure that your server is accessible, worldwide,
+from the internet, by adding the required firewall and port forward rules.
 
 [[install-lets-encrypts-certbot-client]]
-== Install Let’s Encrypt’s Certbot client
+== Install Let’s Encrypt’s Certbot Client
 
-The latest https://certbot.eff.org[Certbot] client can be installed in
-two ways:
+The https://certbot.eff.org[Certbot] client can be installed in following ways:
 
-1.  From source <install_letsencrypt_via_github>.
-2.  With the Ubuntu
-ppa repository <install_letsencrypt_via_ppa_repository>.
-
-[[via-github]]
-=== Via GitHub
-
-....
-sudo apt-get update
-sudo apt-get install -y git
-sudo git clone https://github.com/certbot/certbot /opt/letsencrypt
-....
-
-To run Certbot use the following command:
-
-....
-sudo /opt/letsencrypt/certbot-auto
-....
-
-NOTE: For the sake of simplicity, the path chosen for the installation is `/opt/letsencrypt`. You can use any path that fits your needs. And unless explicitly denied, Certbot will auto-update on each run.
-
-As part of the first run, certbot-auto will install any missing
-dependencies.
+1. Installation via xref:apt[apt install]
+2. With the Ubuntu xref:via_ppa_repository[ppa repository].
 
 [[via-apt]]
-=== Via Apt
+=== Via apt
```

Comment:
Can you revert this change, pls, as the proper spelling is either Apt or APT. See https://wiki.debian.org/Apt.